torch/testing/_internal/common_methods_invocations.py

```diff
@@ -7679,17 +7706,6 @@ def _gather(shape, index_dim, max_indices):
                           args=(1, idx, src, reduce),
                           kwargs={'include_self': True})
 
-
-    # Check that gradients are evenly distributed for amax/amin
-    if requires_grad and (reduce in ['amax', 'amin']):
-        input = _tensor((M, S))
-        src = input.clone().detach_().requires_grad_(requires_grad)
-        idx = torch.arange(S).expand((M, S)).to(dtype=torch.long, device=device)
-
-        yield SampleInput(_tensor((M, S)),
```

Comment:
There was a bug here where I passed a new tensor rather than input so the gradgrad failure wasn't caught.